src/layout/ErrorLayout/ErrorLayout.module.scss

```diff
@@ -2,12 +2,12 @@
 @import 'styles/govBanner';
 @import 'styles/sfds/sfdsDependencies';
 
-$hero-image: url('/assets/images/hero.jpeg');
+$hero-image: url('/assets/images/hero-transparent.png');
```

Comment:
Question: There are a few other files that use the `hero.jpeg` including the `src/styles/_themes.scss` file. Should we update those eventually as well?

Reply:
those other components probably don't want a transparent image, so probably not